winml/test/model/model_tests.cpp

```diff
@@ -227,6 +227,7 @@ static std::vector<ITestCase*> GetAllTestCases() {
   };
   allDisabledTests.insert(std::begin(x86DisabledTests), std::end(x86DisabledTests));
 #endif
+allDisabledTests.insert(ORT_TSTR("cntk_simple_seg"));
```

Comment:
Maybe add a comment here too  and should this change also get a sign-off from the WinML team ?

Reply:
Sounds good. I have just added comments for reference. Since this PR removes a test in WinML (`winml/test/model/model_tests.cpp`), @fdwr it would be super great if this PR can get your approval. Thank you for your time!

Reply:
@jcwchen I'm lightly familiar with them, but this looks fine, as you're just disabling a test that relied on the previous behavior, which as evidently aberrant to the spec. Does the DML EP need to be updated too? (along with CUDA...)

@smk2007 is actually on WinML. So tagging him here too in case he has concerns.